tztrout/data.py

```diff
@@ -275,20 +250,10 @@ def _get_tz_identifiers_for_us_zipcode(self, zipcode):
             else:
                 return
 
-        # Find all the TZ identifiers with a non-DST offset of zipcode.timezone
-        tz_ids = self._get_tz_identifiers_for_offset(
-            'US', datetime.timedelta(hours=zipcode.timezone)
-        )
-
-        # For each of the found identifiers, cross-reference the DST data from
-        # pytz and pyzipcode and only include the identifier if they match
-        ret_ids = []
-        for id in tz_ids:
-            tz = pytz.timezone(id)
-            tz_experiences_dst = self._experiences_dst(tz)
-            if tz_experiences_dst == bool(zipcode.dst):
-                ret_ids.append(id)
-        return ret_ids
+        # Find all the TZ identifiers for a Zipcode given the latitude and longitude
+        # using the TimezoneFinder library
+        tz_ids = tf.timezone_at(lng=zipcode.longitude, lat=zipcode.latitude)
+        return [tz_ids]
```

Comment:
Either the name of the variable is wrong (singular `tz_id`? - comment above also indicates "all identifiers") or the surrounding list shouldn't be there.

Reply:
Fixed in 5e07e1f